apps/user-portal/src/components/overview/widgets/account-status-widget.tsx

```diff
@@ -29,28 +33,236 @@ import { ThemeIcon } from "../../shared";
+        } else if (status === ProfileCompletionStatus.WARNING) {
+            return AccountStatusShields.warning;
+        }
+
+        return AccountStatusShields.good;
+    };
+
+    /**
+     * Generates the more info popup.
+     *
+     * @param attributes - Relevant attributes.
+     * @return {JSX.Element}
+     */
+    const generatePopup = (attributes: any): JSX.Element => (
+        ((attributes.completedAttributes
+            && attributes.completedAttributes.length
+            && attributes.completedAttributes.length > 0)
+            || (attributes.incompleteAttributes
+                && attributes.incompleteAttributes.length
+                && attributes.incompleteAttributes.length > 0))
+            ? (
+                <Popup
+                    trigger={ <Icon color="grey" name="info circle"/> }
+                    position="bottom center"
+                    className="list-content-popup"
+                    hoverable
+                    pinned
+                    content={ (
+                        <>
+                            {
+                                (attributes.completedAttributes
+                                    && attributes.completedAttributes.length
+                                    && attributes.completedAttributes.length > 0)
+                                    ? (
+                                        <>
+                                            <div className="header">
+                                                <Icon color="green" name="check circle"/>
+                                                { t("views:components.overview.widgets.accountStatus." +
+                                                    "completedFields") }
+                                            </div>
+                                            <ul>
+                                                {
+                                                    attributes.completedAttributes
+                                                        .map((attr, index) => (
+                                                            <li key={ index }>{ attr }</li>
+                                                        ))
+                                                }
+                                            </ul>
+                                        </>
+                                    )
+                                    : null
+                            }
+
+                            {
+                                (attributes.incompleteAttributes
+                                    && attributes.incompleteAttributes.length
+                                    && attributes.incompleteAttributes.length > 0)
+                                    ? (
+                                        <>
+                                            <div className="header">
+                                                <Icon color="red" name="cancel"/>
```

Comment:
Shall we add the "times circle" icon here?

Reply:
`times circle` have been added.